pennylane/optimize/momentum_qng.py

```diff
@@ -0,0 +1,171 @@
+        V(\theta_1, \dots, \theta_{i-1}),
+
+    where all parametrized gates can be written of the form :math:`X(\theta_{i}) = e^{i\theta_i K_i}`.
+    That is, the gate :math:`K_i` is the *generator* of the parametrized operation :math:`X(\theta_i)`
+    corresponding to the :math:`i`-th parameter.
+
+    For each parametric layer :math:`\ell` in the variational quantum circuit
+    containing :math:`n` parameters, the :math:`n\times n` block-diagonal submatrix
+    of the Fubini-Study tensor :math:`g_{ij}^{(\ell)}` is calculated directly on the
+    quantum device in a single evaluation:
+
+    .. math::
+
+        g_{ij}^{(\ell)} = \langle \psi_\ell | K_i K_j | \psi_\ell \rangle
+        - \langle \psi_\ell | K_i | \psi_\ell\rangle
+        \langle \psi_\ell |K_j | \psi_\ell\rangle
+
+    where :math:`|\psi_\ell\rangle =  V(\theta_1, \dots, \theta_{i-1})|0\rangle`
+    (that is, :math:`|\psi_\ell\rangle` is the quantum state prior to the application
+    of parameterized layer :math:`\ell`).
+
+    Combining the quantum natural gradient optimizer with the analytic parameter-shift
+    rule to optimize a variational circuit with :math:`d` parameters and :math:`L` layers,
+    a total of :math:`2d+L` quantum evaluations are required per optimization step.
+
+    For more details, see:
+
+        James Stokes, Josh Izaac, Nathan Killoran, Giuseppe Carleo.
+        "Quantum Natural Gradient."
+        `Quantum 4, 269 <https://doi.org/10.22331/q-2020-05-25-269>`_, 2020.
+
+    .. note::
+
+        The QNG optimizer supports using a single :class:`~.QNode` as the objective function. Alternatively,
+        the metric tensor can directly be provided to the :func:`step` method of the optimizer,
+        using the ``metric_tensor_fn`` keyword argument.
+
+        For the following cases, providing ``metric_tensor_fn`` may be useful:
+
+        * For hybrid classical-quantum models, the "mixed geometry" of the model
+          makes it unclear which metric should be used for which parameter.
+          For example, parameters of quantum nodes are better suited to
+          one metric (such as the QNG), whereas others (e.g., parameters of classical nodes)
+          are likely better suited to another metric.
+
+        * For multi-QNode models, we don't know what geometry is appropriate
+          if a parameter is shared amongst several QNodes.
+
+    **Examples:**
+
+    TODO
+
+    .. seealso::
+
+        See the next examples of the Pennylane code, benchmarking the Momentum-QNG optimizer 
+        together with the basic QNG, Momentum and Adam: 
+            QAOA:     https://github.com/borbysh/Momentum-QNG/blob/main/QAOA_depth4.ipynb
+            VQE:      https://github.com/borbysh/Momentum-QNG/blob/main/portfolio_optimization.ipynb
+
+    Keyword Args:
+        momentum=0.9 (float): the user-defined hyperparameter :math:`\rho`
+        stepsize=0.01 (float): the user-defined hyperparameter :math:`\eta`
```

Comment:
Matching the order of the kwargs in `__init__`:
```suggestion
        stepsize=0.01 (float): the user-defined hyperparameter :math:`\eta`
        momentum=0.9 (float): the user-defined hyperparameter :math:`\rho`
```